docs/source/user-guide/tasks/build-packages/compiler-tools.rst

```diff
@@ -0,0 +1,328 @@
+=======================
+Anaconda compiler tools
+=======================
+
+Anaconda 5.0 switched from OS-provided compiler tools to our own toolsets. This
+allows improved compiler capabilities, including better security and
+performance. This page describes how to use these tools and enable these
+benefits.
+
+Compiler packages
+=================
+
+Before Anaconda 5.0, compilers were installed using system tools such as XCode
+or ``yum install gcc``. Now there are conda packages for Linux and macOS
+compilers. These packages are split into separate compilers:
+
+Linux:
+
+* gcc_linux-64
+* gxx_linux-64
+* gfortran_linux-64
+
+macOS:
+
+* clang_osx-64
+* clangxx_osx-64
+* gfortran_osx-64
+
+A compiler's "build platform" is the platform where the compiler runs and
+builds the code.
+
+A compiler's "target platform" is the platform where the built code will run.
```

Comment:
Can we change "target platform" to "host platform"? We use "host" for this everywhere else and I think using "target" for some bits just muddies the water.

Reply:
Oh! Yes, that's much better. I realized that we needed some simple terms for those concepts but didn't realize that host was being used already. (I got the build/target terminology from https://en.wikipedia.org/wiki/Cross_compiler . "The fundamental use of a cross compiler is to separate the build environment from target environment.") I'll edit now and replace build/target with build/host in all cases.

Reply:
To help define "host" in the readers' heads I'd be tempted to go for:

> A compiler's "host platform" is the platform where the built code will finally be hosted/run.

Reply:
I like that. Went ahead and made the change.